### PR TITLE
Minimal requirements file for unit tests

### PR DESCRIPTION
**Pull request by sorensenjs**
(may not work with notebooks)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,71 +1,7 @@
...
-tensorboard==1.14.0
-terminado==0.8.1
-testpath==0.3.1
-tornado==4.5.2
-traitlets==4.3.2
-wcwidth==0.1.7
-Werkzeug==0.15.3
-widgetsnbextension==3.0.2
+tensorflow==1.14.0; python_version < '3.0'
+tensorflow==2.3.1; python_version > '3.0'
```
Comment (idk3) [inline requirements.txt:7]: Why not keep this at 1.15.4 for PY3? IMO easier to maintain this at TF1 for now.

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ language: python
 
 python:
   - "2.7"
+  - "3.8"
```
Comment (idk3) [inline .travis.yml:5]: (Note that if you go with my suggestion in requirements, you'll have to change this to PY3.7, see https://github.com/tensorflow/tensorflow/issues/39768#issuecomment-632799481)

**Review by idk3 — APPROVED** [review]
Two comments -- this is OK to submit as is, but I'm worried the TF1 --> TF2 change will cause headaches later on

File: .travis.yml (hunk 2 above)
Comment (sorensenjs) [reply .travis.yml]: ERROR: Could not find a version that satisfies the requirement tensorflow==1.15.4 (from -r requirements.txt (line 7)) (from versions: 2.2.0rc1, 2.2.0rc2, 2.2.0rc3, 2.2.0rc4, 2.2.0, 2.2.1, 2.3.0rc0, 2.3.0rc1, 2.3.0rc2, 2.3.0, 2.3.1)
ERROR: No matching distribution found for tensorflow==1.15.4 (from -r requirements.txt (line 7))

Note that on my localhost this doesn't work:

+ python model_bias_analysis_test.py
2020-10-19 14:59:35.880364: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib/mesa-diverted/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu/mesa:/usr/lib/x86_64-linux-gnu/dri:/usr/lib/x86_64-linux-gnu/gallium-pipe
2020-10-19 14:59:35.880436: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Traceback (most recent call last):
  File "model_bias_analysis_test.py", line 8, in <module>
    import model_bias_analysis as mba
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/unintended_ml_bias/model_bias_analysis.py", line 28, in <module>
    import matplotlib.pyplot as plt
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/__init__.py", line 62, in pylab_setup
    backend_mod = __import__(backend_name, globals(), locals(),
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/backend_tkagg.py", line 4, in <module>
    from . import tkagg  # Paint image to Tk photo blitter extension.
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/tkagg.py", line 5, in <module>
    from six.moves import tkinter as Tk
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 92, in __get__
    result = self._resolve()
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 115, in _resolve
    return _import_module(self.mod)
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 82, in _import_module
    __import__(name)
ModuleNotFoundError: No module named 'tkinter'

**Pull request merged**